### PR TITLE
Zybooks 7.7 function

### PR DESCRIPTION
**Pull request by eshagupta01**

### CODE REVIEW

**Comment by harshaucsb** [comment]
Can you check the case where you pass multiple strings in the list and also maybe an integer as the first element too - something like - [1, 2, 3, "ax", "by", 7] .. its okay to have 4/5 asserts too for a question

File: tests/correct.py [hunk 1]
```diff
@@ -60,5 +60,9 @@ def modify_list(num_list):
     assert get_dictionary_value({'a': 1, 'b': 2, 'c': 3}, 'd') == 'The available options are - a, b, c'
 
     assert modify_list([1, 2, 3, 4, 5]) == [1, 3, 5, 7, 9]
+    assert modify_list([1]) == [1]
     assert modify_list(['hello', 10]) == 'List contains type that is not an integer at position 0'
     assert modify_list([]) == -1
+    assert modify_list([0.1, 'Kelly', 'h']) == 'List contains type that is not an integer at position 0'
```
Comment (harshaucsb) [inline tests/correct.py:66]: We need to have these assertions as part of the test because in the test we check the student's solution against these cases.

Comment (eshagupta01) [reply tests/correct.py:66]: Oh right, sorry let me fix that

File: tests/config.py [hunk 2]
```diff
@@ -35,6 +35,7 @@
 
 fun_names_dict = {
     'get_dictionary_value': 'get_dictionary_value',
+    'modify_list': 'modify_list'
```
Comment (harshaucsb) [inline tests/config.py]: can you add a comma to the end here like "'modify_list': 'modify_list'," as the other PRs will not clash with yours in that case and there won't be any merge conflicts

**Review by harsha-miryala — APPROVED** [review]

**Pull request merged**